apollo-router/src/query_planner/bridge_query_planner.rs

```diff
@@ -236,27 +224,6 @@ impl PlannerMode {
         Ok(Arc::new(planner))
     }
 
-    async fn js_introspection(
-        &self,
-        sdl: &str,
-        configuration: &Configuration,
-        old_js_planner: &Option<Arc<Planner<QueryPlanResult>>>,
-        cache: CacheStorage<String, Response>,
-    ) -> Result<Arc<Introspection>, ServiceBuildError> {
-        let js_planner = match self {
-            Self::Js(js) => js.clone(),
-            Self::Both { js, .. } => js.clone(),
-            Self::Rust(_) => {
-                // JS "planner" (actually runtime) was not created for planning
-                // but is still needed for introspection, so create it now
-                Self::js_planner(sdl, configuration, old_js_planner).await?
-            }
-        };
```

Comment:
> If introspection is now completely in rust, does that mean v8 won't start up with just `experimental_query_planner_mode: new`? 

Correct. This is the relevant code change. Here the badly-named `js_planner` represents a V8 runtime.

I didn’t realize we publish docs from `next`, I’ll make a separate PR